### DISCUSSION TITLE
LlamaIndex usage for generating knowledge-databases (apart from Chat with Docs)

### DISCUSSION THREAD

**Discussion by d13g4** [ParisNeo/lollms-webui#340] [Ideas]
Wouldn't that be a nice thing to have? :) 
https://gpt-index.readthedocs.io/en/latest/index.html

**Comment by ParisNeo** [comment]
I was thinking of this. I actually have a similar thing that I coded myself and it is integrated in lollms v6.0 and works with all non scripted personalities. If you go to settings/main settings, you'll find all the details and options about that.

Maybe I'll add llamaindex support as an option.
